app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/bakeManifestConfig.html

```diff
@@ -0,0 +1,40 @@
+<div class="form-horizontal" xmlns="http://www.w3.org/1999/html">
+  <div class="alert alert-warning">
+    <strong>Warning!</strong> This stage is under active development and is subject to change.
+  </div>
+  <h4>Template Renderer</h4>
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">
+      Render Engine
+      <help-field key="pipeline.config.bake.manifest.templateRenderer"></help-field>
+    </div>
+    <div class="col-md-3">
+      <select class="form-control input-sm"
+              ng-model="ctrl.$scope.stage.templateRenderer">
+        <option ng-repeat="sessionAffinity in ctrl.templateRenderers"
```

Comment:
Looks like `sessionAffinity` is copy-paste left over from the "Session Affinity" input; should probably change the name here